Microsoft.O365.Security.Native.ETW/EventRecordMetadata.hpp

```diff
@@ -153,6 +153,68 @@ namespace Microsoft { namespace O365 { namespace Security { namespace ETW {
             return dest;
         }
 
+#pragma endregion
+
+#pragma region ExtendedData
+
+        /// <summary>
+        /// If the event's extended data contains an Argon container ID, retrieve it.
```

Comment:
rather than Argon, could we use the public-facing term for this level of container isolation? https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container

Reply:
Sounds good, I've updated those references to "Windows container ID" and references to process isolation.